packages/mui-material/src/styles/ThemeProvider.tsx

```diff
@@ -57,6 +57,12 @@ export interface ThemeProviderProps<Theme = DefaultTheme> extends ThemeProviderC
    * @default 'mui-color-scheme'
    */
   colorSchemeStorageKey?: string;
+  /*
+   * If `true`, ThemeProvider will not rerender and the initial value of `mode` comes from the local storage.
+   * For SSR applications, you must ensure that the server render output must match the initial render output on the client.
```

Comment:
Should we rewrite this prop description?

- "If `true`, ThemeProvider will not rerender and the initial value of `mode` comes from the local storage." I don't understand what this means. Is this the same as https://mui.com/material-ui/react-use-media-query/#api?
- must […] must.